src/pages/Menu/Menu.tsx

```diff
@@ -30,20 +31,22 @@ export const MenuPage = (): ReactElement => {
       <Helmet>
         <title>Menu - Answer King</title>
       </Helmet>
-      <MenuCategories selectedCategory={selectedCategory} setSelectedCategory={setSelectedCategory} categories={categories.data} />
-      <div className="mt-10 grid grid-cols-12">
-        <div className="col-span-1"></div>
-        <div className="col-span-8 text-center">
-          <MenuItems
-            products={products.data.filter(
-              (product) => product.retired === false && product.categories?.find((categoryId) => categoryId === selectedCategory)
-            )}
-          />
+      <PageTransition>
+        <MenuCategories selectedCategory={selectedCategory} setSelectedCategory={setSelectedCategory} categories={categories.data} />
+        <div className="mt-10 grid grid-cols-12">
+          <div className="col-span-1"></div>
+          <div className="col-span-8 text-center">
+            <MenuItems
+              products={products.data.filter(
+                (product) => product.retired === false && product.categories?.find((categoryId) => categoryId === selectedCategory)
```

Comment:
does `!product.retired` work here? it's just a boolean so it shouldn't ever be anything but true or false, right?

Reply:
looks like harry us doing a [strict equals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) `===` , if he did `!product. retired`, that's going to be a [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) check.

Not sure if this makes a difference, I guess the former is safer though, and no [type coercion](https://developer.mozilla.org/en-US/docs/Glossary/Type_coercion) needs to take place.